setup.cfg

```diff
@@ -15,7 +15,7 @@ github_project = spacetelescope/jdaviz
 zip_safe = False
 packages = find:
 include_package_data = True
-python_requires = >=3.8
+python_requires = >=3.9
```

Comment:
If a user has python 3.8 and installs or attempts to upgrade jdaviz via pip, will they then just get the most recent release _before this PR_?  If so, I'm concerned that this might just result in users not getting bugfixes and new features (either intentionally because they don't want to update their python environment or just because they don't realize they need to update in order to get newer releases).

If our minimum requirements on upstream packages do not require 3.9 (yet), then would we be better off holding off as long as possible?

Reply:
I understand the concern, but with both `numpy` and `astropy` already requiring 3.9, these users already wouldn't be getting new features in relevant packages (I assume astropy LTS still supports 3.8 for bug fixes at least). We can make the python requirement change extra prominent in the 3.5 release notes, but I think it's time for this. Users stuck on 3.8 would still get bug fix releases for 3.4, but also I'm skeptical that users who haven't upgraded from 3.8 are upgrading their packages either.